ego4d/internal/validate.py

```diff
@@ -170,6 +175,7 @@ def _validate_mp4(
         error_message: List[ErrorMessage]: a list to store ErrorMessage objects
         generated when validating mp4 files in s3.
     """
+    print("validating mp4")
     for video_id, video_infos in video_info_dict.items():
```

Comment:
Can we wrap this loop with tqdm and do the same for other loops? I think this loop takes a while to execute if I'm not mistaken.